### PR TITLE
Support release model repo using kaniko on kubernetes

### PR DESCRIPTION
**Pull request by typhoonzero**

### CODE REVIEW

File: pkg/modelzooserver/templates.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ package modelzooserver
 
 const sampleInitCode = `from .my_test_model import DNNClassifier`
 
-const sampleDockerfile = `FROM sqlflow/sqlflow
+const sampleDockerfile = `FROM ubuntu
```
Comment (typhoonzero) [inline pkg/modelzooserver/templates.go]: Changed to `ubuntu` for faster testing.

Comment (Yancey0623) [reply pkg/modelzooserver/templates.go]: Better to use a fixed ubuntu version.

File: pkg/modelzooserver/image_build.go [hunk 2]
```diff
@@ -104,3 +109,114 @@ func buildAndPushImage(dir, name, tag string, dryrun bool) error {
...
+	if err != nil {
+		return err
+	}
+	defer os.Chdir(cwd)
+	if err := os.Chdir(dir); err != nil {
+		return err
+	}
+
+	destination := fmt.Sprintf("%s:%s", name, tag)
+	podTemplate := fmt.Sprintf(`'{
```
Comment (Yancey0623) [inline pkg/modelzooserver/image_build.go:162]: Using Go template would be better to understand.

File: pkg/modelzooserver/image_build.go [hunk 3]
```diff
@@ -16,13 +16,18 @@ package modelzooserver
...
 	"io/ioutil"
 	"net/http"
 	"os"
 	"os/exec"
 	"strings"
+
+	"sqlflow.org/sqlflow/pkg/randstring"
 )
 
+const kanikoImage = "registry.cn-hangzhou.aliyuncs.com/sql-machine-learning/kaniko-executor"
```
Comment (Yancey0623) [inline pkg/modelzooserver/image_build.go]: Maybe we can use the official Dockerhub registry to be the default value, and specify another registry by an environment variable.

File: pkg/modelzooserver/image_build.go (hunk 2 above)
Comment (typhoonzero) [reply pkg/modelzooserver/image_build.go:162]: It's just simple string replace here, use `Sprintf` is enough.

File: pkg/modelzooserver/image_build.go (hunk 3 above)
Comment (typhoonzero) [reply pkg/modelzooserver/image_build.go]: Changed to use env, yet I kept to use the Aliyun image by default.

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**